website/content/about/CloudNativeSustainabilityWeek.md

```diff
@@ -0,0 +1,80 @@
+---
+title: CNCF Cloud Native Sustainability Week
+description: Cloud Native Sustainability Week is an event to organize meetings in CNCF community groups around the theme of Cloud Native Sustainability during the second week of October 2023.
+slug: cloud-native-sustainability-week
+---
+
+The CNCF Global Week of Cloud Native Sustainability is an event organized by the [Cloud Native Computing Foundation (CNCF)](http://cncf.io) community to address the emerging topic of environmental sustainability in the cloud native industry and open-source space. The event aims to engage with the community and industry, understand the current landscape, and promote collaboration and knowledge sharing.
```

Comment:
Change to "open source" to comply with naming convention.
Suggestion: "The event aims to engage with the community and IT industry as a whole, get a better understanding of  the current environmental sustainability landscape, and promote collaboration and knowledge sharing on the topic."